qiskit_addon_dice_solver/dice_solver.py

```diff
@@ -355,7 +355,8 @@ def _write_input_files(
     # We don't want Dice to be noisyu for now so we hard code noio
     noio = "noio\n"
     # The number of determinants to write as output. We always want all of them.
-    dim = math.comb(norb, num_up) * math.comb(norb, num_dn)
+    # The larger integer in C++ is 2147483647
```

Comment:
The ints should be unsigned, which would mean they could go up to 2**32.

Reply:
I am incorrect here. They use `int`, which is signed; however, it could be 64 bit depending on architecture. Let's use ctypes here

Reply:
https://github.com/caleb-johnson/Dice/blob/9119fd71a7ab4567e1e731fc03ff17511c3bb881/SHCI/SHCI.cpp#L295